### ISSUE TITLE
Enum class generator has invalid autoload file path

### ISSUE THREAD

**Issue by tomaszkane** [thunderer/Platenum#4]
1. `cd myProjectDir`
2. `composer update`
3. `./vendor/thunderer/platenum/bin/generate`

returns PHP Warning and Error:
```
require(): Failed opening required '/home/kane/workspace/FOO/vendor/thunderer/platenum/vendor/autoload.php' (include_path='.:/usr/share/php') in /home/kane/workspace/FOO/vendor/thunderer/platenum/bin/generate on line 10
```
Line 10: `$vendorLoaderPath  = dirname(__DIR__, 3).'/vendor/autoload.php';` in genrator returns path: `/home/kane/workspace/FOO/vendor/vendor/autoload.php`.
We have double vendors here :-) ?

**Comment by thunderer** [comment]
@tomaszkane Yes, `dirname()` should go back `4` times, not `3`. This should be fixed in #5, let me know if it works for you. I already merged #3.

**Comment by tomaszkane** [comment]
generator work, but default return help is invalid?
```bash
 ./vendor/thunderer/platenum/bin/generate
Platenum (c) 2019 Tomasz Kowalczyk.

usage: bin/generate source class keys,values
       bin/generate constants MyEnum key1,key2=value2,key3
```
'usage' should be describe as three commands like in README?

**Comment by thunderer** [comment]
@tomaszkane Can you be more specific? What exactly is invalid there? The first line is a definition, the second is an example. I see that this may be a little unclear - the idea to make examples for all available sources is a good one, I'll include that in #5.

**Comment by tomaszkane** [comment]
So maybe do it like Yii2 framework does:
```bash
    bin/generate <source> <class> [...keys, values...]
    bin/generate constants MyEnum key1=value1,key2=value2
```
?

**Comment by thunderer** [comment]
@tomaszkane I did exactly that in the latest version of #5, including real-world examples for all sources. Is there anything else that can be improved from your point of view?

**Comment by thunderer** [comment]
@tomaszkane I just merged #5, can we close this issue?

**Comment by tomaszkane** [comment]
Perfect.
